pkgs/by-name/oc/octodns/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "octodns";
+  version = "1.2.1";
+  pyproject = true;
+
+  disabled = python3Packages.pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "octodns";
+    repo = "octodns";
+    rev = "v${version}";
+    hash = "sha256-oZB9Wb9wWOy8C8FJEDk449OtgLfdcIi3vIzW5AV6XNA=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.setuptools
+    python3Packages.wheel
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    dnspython
+    fqdn
+    idna
+    natsort
+    python-dateutil
+    pyyaml
+    six
```

Comment:
```suggestion
```
I think nix-init picked up `six` from requirements.txt, but we should not need it.
https://github.com/octodns/octodns/blob/cac964c91b89bc4fec718cdb0c098db863cc3c71/setup.py#L89-L96